README.md

```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
+
+The turbulent nature of the urban flow field and the inherent heterogeneity and wide range of scales associated with the urban environment result in a complex modelling problem. Large-eddy simulation (LES) is an approach to turbulence modelling used in computational fluid dynamics to simulate turbulent flows over a wide range of spatial and temporal scales. LES is one of the most promising tools to model the interactions typical of urban areas due to its ability to resolve the urban flow field at resolutions of _O_(1 m, 0.1 s), over spatial domains of _O_(100 m), and time periods of _O_(1 h). Although there are many scalable LES models for atmospheric flows, to our knowledge, only few are capable of explicitly representing buildings and of modelling the full range of urban processes (e.g. PALM-4U; Resler et al. 2017; Maronga et al. 2020; or OpenFoam; Weller et al. 1998)..
+
+uDALES is capable of modelling urban environments at the microclimate scale including wet thermodynamics, idealized and complex morphologies, three-dimensional urban surface energy balance and real-time pollution dispersion.
+
+## Publications and validation
+
+To-date, uDALES has been used to study a number of research questions:
+
+| Research application                                         | Reference                                                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------------------------------- |
+| Urban boundary layers/ boundary-layer meteorology            | Grylls et al. (2019); Sutzl et al. (2020)                                       |
+| Urban climate (radiation, green roofs and walls, trees etc.) | Suter (2018); Suter et al. (2021); Grylls & van Reeuwijk (2021)                 |
+| Pollution dispersion/ urban air quality                      | Grylls et al. (2019);  Grylls (2020)                                            |
+| Buoyancy/ convective and stable conditions                   | Suter (2018); Grylls et al. (2019); Grylls (2020); Grylls & van Reeuwijk (2021) |
+
+uDALES has been validated against both field and experimental studies and evaluated against other numerical models. The dynamic core, DALES, has been used to study atmospheric processes for over 30 years, being validated extensively in the process and used in several atmospheric intercomparison studies (Nieuwstadt and Brost 1986; Heus et al. 2010). The use of the immersed boundary method to model obstacles was validated by Tomas et al. (2016). The wall functions and three-dimensional surface energy balance model was validated by Suter (2018). The ability to model pollutuon dispersion in both idealised and realistic morphologies was been validated by Grylls et al. (2019) and Grylls (2020).
```

Comment:
maybe a direct link to the DALES repository? The dynamic core, [DALES](https://github.com/dalesteam/dales),

Reply:
Done!